.github/workflows/pr-validation.yml

```
@@ -0,0 +1,189 @@
+name: PR Validation
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)